ds_judgements_public_ui/sass/includes/_govuk_overrides.scss

```diff
@@ -193,35 +193,29 @@
 
 .govuk-header__link--homepage:hover,
 .govuk-header__link--homepage:active {
-  border-bottom: $space-1 solid $color-black;
+  border-bottom: 0.25rem solid $color-black;
 }
 
-.govuk-header__navigation-item--active a,
-.govuk-header__navigation-item--active a:hover,
-.govuk-header__navigation-item--active a:visited {
+.govuk-header__navigation-item--active .govuk-header__link,
+.govuk-header__navigation-item--active .govuk-header__link:hover,
+.govuk-header__navigation-item--active .govuk-header__link:visited {
   color: $color-yellow;
-  font-weight: bold !important;
-
+  font-weight: bold;
+  border-bottom: 0.25rem solid $color-yellow;
+  text-decoration: none;
   &:focus {
     background-color: transparent;
     box-shadow: none;
   }
 }
 
-.govuk-header__navigation-item--active .govuk-header__link,
-.govuk-header__navigation-item .govuk-header__link:hover {
-  color: $color-yellow;
-  font-weight: bold;
-  border-bottom: $space-1 solid $color-yellow;
-  text-decoration: none;
-}
-
 .govuk-header__navigation-item .govuk-header__link {
```

Comment:
I think we need to add a focus style somewhere around here as when you click a nav item it goes black and we can't see it anymore. Also the hover affect has disappeared where it goes yellow and bold.

Reply:
@jlhdxw checked with Terry and hover should only be taking away the underline not making it bold yellow so updated it thanks for spotting